app/src/main/AndroidManifest.xml

```diff
@@ -21,7 +21,7 @@
         android:label="@string/app_name"
         android:theme="@style/AppTheme"
         android:vmSafeMode="true">
-
+        <activity android:name=".chat.search.ChatSearchActivity"></activity>
         <activity
```

Comment:
I guess it is not necessary. If it is, could you please justify why?

Reply:
I have created an activity for displaying the result. And when activity is created, its declaration is automatically added in the manifest file.